easybuild/easyconfigs/o/OTF2/OTF2-3.0.3-GCCcore-13.2.0.eb

```diff
@@ -34,17 +35,22 @@ builddependencies = [
 dependencies = [
     # SIONlib container support (optional):
     ('SIONlib', '1.7.7', '-tools'),
+    # OTF2 Python bindings
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
 ]
 
+local_pyshortver = '3.11'
 configopts = '--enable-shared'
 
+modextrapaths = {'PYTHONPATH': ['lib64/python%s/site-packages' % local_pyshortver]}
 
 sanity_check_paths = {
     'files': ['bin/otf2-config', 'include/otf2/otf2.h',
               'lib/libotf2.a', 'lib/libotf2.%s' % SHLIB_EXT],
     'dirs': [],
 }
-
-sanity_check_commands = ['%(namelower)s-config --help']
+sanity_check_commands = ['%(namelower)s-config --help',
+                         'python -c "import %(namelower)s"']
```

Comment:
I would suggest to [use `-s`](https://docs.python.org/3/using/cmdline.html#cmdoption-s) here as otherwise the $HOME packages might be used causing failures/success:

```diff
-                         'python -c "import %(namelower)s"']
+                         'python -s -c "import %(namelower)s"']
``````

Reply:
@Flamefire This is relevant for several easyblocks too where we use `python -c`?

Reply:
Yes it is

Reply:
-> https://github.com/easybuilders/easybuild-easyblocks/pull/3440